python/ray/_private/utils.py

```diff
@@ -1964,3 +1970,22 @@ def validate_node_labels(labels: Dict[str, str]):
                 f"{ray_constants.RAY_DEFAULT_LABEL_KEYS_PREFIX}. "
                 f"This is reserved for Ray defined labels."
             )
+
+
+def pasre_pg_formatted_resources_to_origin(pg_formatted_resources):
+    origin_resources = {}
+    if pg_formatted_resources is None:
+        return origin_resources
+
+    for key, value in pg_formatted_resources.items():
+        result = PLACEMENT_GROUP_RESOURCE_PATTERN.match(key)
+        if result and len(result.groups()) == 2:
+            origin_resources[result.group(1)] = value
+            continue
+        result = PLACEMENT_GROUP_RESOURCE_BUNDLED_PATTERN.match(key)
+        if result and len(result.groups()) == 3:
```

Comment:
I feel like we don't need this part of logic? Don't we always allocate wildcard resources? (so this is always equivalent to the logic above)

Reply:
Deleting this logic is also possible. The code can be simplified.

Reply:
hmm does it work if we have this kind of case?

```
CPU_group_4da1c24ac25bec85bc817b258b5201000000: 2.0
CPU_group_0_4da1c24ac25bec85bc817b258b5201000000: 1.0
CPU_group_1_4da1c24ac25bec85bc817b258b5201000000: 1.0
```

Reply:
we can just ignore them?

Reply:
Same as above, ignore or  overwrite is same.